test/index.test.ts

```diff
@@ -11,6 +11,7 @@ import { ethers } from "hardhat";
 
 import { ToucanClient } from "../dist";
 import { IToucanCarbonOffsets } from "../dist/typechain";
+import { PoolSymbol } from "../dist/types";
```

Comment:
can you create a follow-up PR to test the typescript code instead of generated js code?

Reply:
I am also in favor of testing the code that is shipped.

Reply:
![d13f-3f03-4401-a0c1-c5617f634e9e](https://user-images.githubusercontent.com/16206423/182118646-8ffde79e-3901-4de9-ba08-91f166c6d040.jpg)
hehe, it's just I just haven't seen anybody test generated code before. it's like you don't trust `tsc` to do its job right :D I'm not against it, though 😊